samples/bluetooth/direct_test_mode/src/transport/dtm_uart_twowire.c

```diff
@@ -0,0 +1,887 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <errno.h>
+#include <zephyr/drivers/uart.h>
+#include <zephyr/logging/log.h>
+#include <zephyr/kernel.h>
+#include <nrfx_timer.h>
+
+#include "dtm_uart_twowire.h"
+#include "dtm.h"
```

Comment:
You can use:
`#include <dtm.h>`
src directory is a cmake target include directory